.github/workflows/build.yml

```diff
@@ -145,6 +149,14 @@ jobs:
             **/build/reports/tests/*
           compression-level: 9
           if-no-files-found: ignore
+      - name: Archive JUnit XML
+        uses: actions/upload-artifact@v4
+        with:
+          name: junit-xml-${{ matrix.java }}
+          path: |
+            build/junit-xml/**/*.xml
```

Comment:
Should we export the artifact url for this task?

Reply:
I'm not sure we'll keep this step in long term since it is ~50Mb compressed (~1GB decompressed!).